packages/@aws-cdk/aws-codedeploy/lib/deployment-group.ts

```diff
@@ -404,6 +434,31 @@ export class ServerDeploymentGroup extends ServerDeploymentGroupRef {
     }
     return tagsInGroup;
   }
+
+  private autoRollbackConfig(autoRollbackConfig: IAutoRollbackConfig):
+      cloudformation.DeploymentGroupResource.AutoRollbackConfigurationProperty | undefined {
+    const events = new Array<string>();
+    if (this.optionalBoolDefaultTrue(autoRollbackConfig.failedDeployment)) {
+      events.push('DEPLOYMENT_FAILURE');
+    }
+    if (this.optionalBoolDefaultTrue(autoRollbackConfig.stoppedDeployment)) {
+      events.push('DEPLOYMENT_STOP_ON_REQUEST');
+    }
+    if (this.optionalBoolDefaultTrue(autoRollbackConfig.deploymentInAlarm)) {
+      events.push('DEPLOYMENT_STOP_ON_ALARM');
+    }
+
+    return events.length > 0
+      ? {
+        enabled: true,
+        events,
+      }
+      : undefined;
+  }
+
+  private optionalBoolDefaultTrue(value?: boolean): boolean {
```

Comment:
Extract as a free-floating function instead of a method

Reply:
This method actually got completely removed in the latest iteration.